components/Carousel/Carousel.js

```diff
@@ -11,4 +13,7 @@ let carousel = document.querySelector();
     4. Those buttons are gonna need some click handlers.
     5. Think of how you would animate this compoennt. Make the cards slide in and out, or fade. It's up to you!
     6. Have fun!
-*/
\ No newline at end of file
+*/
+const carousel = document.querySelector('.carousel').forEach( carousel => new Carousel(carousel));
+const leftButton = document.querySelector('.left-button');
+const rightButton = document.querySelector('.right-button');
```

Comment:
Challenge yourself to implement this carousel, it will be something you can use in future web pages you make :)